### PR TITLE
Reference new DOT in FAQ

### LINKED ISSUE #829: Change New DOT to DOT
dieter
surag: obi qinwen It's time to ask everyone to change from 'New DOT' to 'DOT'
Please let all exchanges, ecosystem projects, CMC, Gecko,etc. that they are free to go back to the normal ticker

We should reference "New DOT" in the FAQ and/or glossary in case people are looking for it, though.

### PR DESCRIPTION
**Pull request by krichard410**
Closes #829

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -159,6 +159,13 @@ be bridges between Polkadot and most of the other major chains.
 
 ## DOT
 
+### What is the difference between DOT (old) and new DOT?
+
+The DOT (old) unit on Polkadot was at twelve decimal places, otherwise known as 1e12 Plancks. On 21
+August, 2020, Demoniation Day, the DOT (old) value was redominated to 1e10 Plancks, meaning that the
```
Comment (laboon) [inline docs/faq.md]:
```suggestion
August, 2020, Denomiation Day, the DOT (old) value was redenominated to 1e10 (10'000'000'000, or ten billion) Plancks, meaning that the
```

File: docs/faq.md [hunk 2]
```diff
@@ -159,6 +159,13 @@ be bridges between Polkadot and most of the other major chains.
...
+new DOT was valued at ten decimal places. Following the redomination, the new DOT changed back to
```
Comment (laboon) [inline docs/faq.md]:
```suggestion
new DOT was valued at ten decimal places. Following the redenomination, the new DOT changed back to
```

File: docs/faq.md [hunk 3]
```diff
@@ -159,6 +159,13 @@ be bridges between Polkadot and most of the other major chains.
...
+DOT.
```
Comment (laboon) [inline docs/faq.md]: It didn't "change back" to DOT, we started _calling_ it DOT instead of _New DOT_.

Comment (laboon) [inline docs/faq.md]: Include link to the Redenomination page here.

**Review by laboon — CHANGES_REQUESTED** [review]

**Comment by laboon** [comment]
Kirsten -
   Looks better but just realized that there are a few other places where new DOT are referenced.  We should call all DOT just "DOT" (_not_ New DOT) except on the redenomination page, FAQ, or anywhere else we are specifically referring to the redenomination.

```
(1094) ~/web3/polkadot-wiki/docs $ grep -i "New Dot" *.md
build-protocol-info.md:redenomination occurs, we recommend that you use the symbol "_New DOT_" to clearly indicate that you
build-protocol-info.md:> redenomination from its original sale. New DOT are 100x smaller than DOT (old). Therefore, your
build-protocol-info.md:example, when a user claims their DOT from the Ethereum DOT indicator contract to a new DOT address,
learn-DOT.md:(also known as _New DOT_) are equal to 1e10 Planck.
learn-accounts.md:(on Kusama) or 0.01 DOT (old) / 1 _New DOT_ (on Polkadot mainnet) .
```

**Comment by laboon** [comment]
@krichard410 ^

**Comment by krichard410** [comment]
@laboon I changed the "learn-dot" and the "learn-accounts" pages, the "build-protocol-info" page had the three new DOT reference to the redenomination.

File: docs/learn-DOT.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ The smallest unit of account in a Substrate network (Polkadot, Kusama, etc.) is
 reference to [Planck Length](https://en.wikipedia.org/wiki/Planck_length), the smallest possible
 distance in the physical Universe). You can compare the Planck to satoshis or wei, while the DOT is
 like a bitcoin or an ether. Kusama tokens (KSM) are equal to 1e12 Planck, and Polkadot mainnet DOT
-(also known as _New DOT_) are equal to 1e10 Planck.
+(also known as _DOT_) are equal to 1e10 Planck.
```
Comment (laboon) [inline docs/learn-DOT.md]:
```suggestion
are equal to 1e10 Planck.
```

Comment (laboon) [reply docs/learn-DOT.md]: Doesn't make sense to say DOT (also known as _DOT_) =)

File: docs/learn-accounts.md [hunk 5]
```diff
@@ -171,7 +171,7 @@ Hardware wallet integration from [Ledger](https://ledger.com) is coming soon.
 
 When you generate an account (address), you only generate a _key_ that lets you access it. The
 account does not exist yet on-chain. For that, it needs the existential deposit: 0.001666666667 KSM
-(on Kusama) or 0.01 DOT (old) / 1 _New DOT_ (on Polkadot mainnet) .
+(on Kusama) or 0.01 DOT (old) / 1 _DOT_ (on Polkadot mainnet) .
```
Comment (laboon) [inline docs/learn-accounts.md]:
```suggestion
(on Kusama) or 1 DOT (on Polkadot mainnet) .
```

**Review by laboon — CHANGES_REQUESTED** [review]

**Pull request merged**